test/compilable/extra-files/ddocAny-postscript.sh

```diff
@@ -3,6 +3,6 @@
 source tools/common_funcs.sh
 
 grep --text -v "Generated by Ddoc from" ${OUTPUT_BASE}.html > ${OUTPUT_BASE}.html.2
-diff -pu --strip-trailing-cr ${EXTRA_FILES}/${TEST_NAME}.html ${OUTPUT_BASE}.html.2
+diff -up --strip-trailing-cr ${EXTRA_FILES}/${TEST_NAME}.html ${OUTPUT_BASE}.html.2
```

Comment:
are these change necessary? (ditto throughout)

Reply:
Yes, it is a regression introduced by this FreeBSD version. See https://github.com/dlang/phobos/pull/8156#issuecomment-874872481 for context.